tests/dummy/public/test-forms/lang-strings/data.ttl

```diff
@@ -0,0 +1,5 @@
+@prefix ext: <http://mu.semte.ch/vocabularies/ext/> .
+
+<http://ember-submission-form-fields/source-node> ext:title "Foo".
+<http://ember-submission-form-fields/source-node> ext:title "Bar"@nl.
+<http://ember-submission-form-fields/source-node> ext:title "Baz"@en.
```

Comment:
Not sure if this is the correct way to define language tagged literals, but those are the only examples I found 😄.